### ISSUE TITLE
simulation setup problem

### ISSUE THREAD

**Issue by citaer** [ethz-asl/glocal_exploration#27]
Hello! I follow the simulation setup and go to step 3. Using the command of "roslaunch unreal_airsim parse_config_to_airsim.launch". But the terminal tells me as follows.

```
ROS_MASTER_URI=http://localhost:11311

process[config_parser-1]: started with pid [32258]
/home/cita/0project/simulation_exploration_ws/src/unreal_airsim/app/tools/parse_config_to_airsim.py:19: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if self.target_file_path is "":
[INFO] [1635495743.301164, 0.000000]: 
************************************************************************
* Starting config parser for unreal_airsim to AirSim config conversion *
************************************************************************
[INFO] [1635495743.308029, 0.000000]: Read source: '/home/cita/0project/simulation_exploration_ws/src/unreal_airsim/cfg/demo.yaml'
Traceback (most recent call last):
  File "/home/cita/0project/simulation_exploration_ws/src/unreal_airsim/app/tools/parse_config_to_airsim.py", line 279, in <module>
    parser.parse()
  File "/home/cita/0project/simulation_exploration_ws/src/unreal_airsim/app/tools/parse_config_to_airsim.py", line 78, in parse
    self.setup_target_file()
  File "/home/cita/0project/simulation_exploration_ws/src/unreal_airsim/app/tools/parse_config_to_airsim.py", line 192, in setup_target_file
    old_cfg = json.load(json_file)
  File "/home/cita/anaconda3/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/home/cita/anaconda3/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/home/cita/anaconda3/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/cita/anaconda3/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
REQUIRED process [config_parser-1] has died!
process has died [pid 32258, exit code 1, cmd /home/cita/0project/simulation_exploration_ws/src/unreal_airsim/app/tools/parse_config_to_airsim.py __name:=config_parser __log:=/home/cita/.ros/log/67c79bd0-388e-11ec-b0a6-a4bb6dc91cb0/config_parser-1.log].
log file: /home/cita/.ros/log/67c79bd0-388e-11ec-b0a6-a4bb6dc91cb0/config_parser-1*.log
Initiating shutdown!
[config_parser-1] killing on exit
shutting down processing monitor...
... shutting down processing monitor complete
done
```

Could you please give me some suggestion about this problem?

**Comment by sz990707** [comment]
Did you solve that problem?

**Comment by Schmluk** [comment]
Hi @citaer and @sz990707 

Apologies for the late reply. Apparently this is a python3 problem (code was developed for ROS which uses python2). Do you use python-is-python 3 or similar? We plan to add python3 support in the future.

**Comment by Schmluk** [comment]
The issue should be fixed with [#26](https://github.com/ethz-asl/unreal_airsim/pull/26).
